README.md

```diff
@@ -26,31 +26,29 @@ Depending on your system's configuration, you may need to run this command with
 
 ## Modes
 
-`knife-azure 1.6.0` onwards, we are adding support for Azure Resource Manager. You can easily switch between the
+`knife-azure 1.6.0` onwards supports Azure Resource Manager (preferred). You can easily switch between:
 
-- Service management: commands using the Azure service management API
 - Resource manager: commands using the Azure Resource Manager API
+- Service management: commands using the Azure service management API
 
-They are not designed to work together. Commands starting with `knife azure` use ASM mode, while commands starting with `knife azurerm` use ARM mode.
-
-PLEASE NOTE that `Azuererm` subcommands are experimental and of alpha quality. Not suitable for production use. Please use ASM subcommands for production.
+They are not designed to work together. Commands starting with `knife azurerm` use ARM mode, while commands starting with `knife azure` use ASM mode.
 
 ## Configuration
 
+1. [ARM Configuration] (docs/configuration.md#arm-mode)
 1. [ASM Configuration] (docs/configuration.md#asm-mode)
-2. [ARM Configuration] (docs/configuration.md#arm-mode)
 
 ## Detailed Usage
 
+1. [ARM Mode] (docs/ARM.md)
```

Comment:
Please refer my comment above about the broken link.

Reply:
https://github.com/chef/knife-azure/pull/457/commits/ff750bcca8a576a5916ac61319dfbc6035294c6d